forge-gui/res/cardsfolder/upcoming/titanias_command.txt

```diff
@@ -2,12 +2,12 @@ Name:Titania's Command
 ManaCost:4 G G
 Types:Sorcery
 A:SP$ Charm | Choices$ DBExileAll,DBSearch,DBToken,DBCounter | CharmNum$ 2
-SVar:DBExileAll:DB$ ChangeZoneAll | Origin$ Graveyard | Destination$ Exile | ValidTgts$ Player | TgtPrompt$ Select target player | ChangeType$ Card | RememberedChanged$ True | SubAbility$ DBGainLife | SpellDescription$ Exile target player's graveyard. You gain 1 life for each card exiled this way.
+SVar:DBExileAll:DB$ ChangeZoneAll | Origin$ Graveyard | Destination$ Exile | TgtPrompt$ Select target player | ChangeType$ Card | RememberedChanged$ True | SubAbility$ DBGainLife | SpellDescription$ Exile target player's graveyard. You gain 1 life for each card exiled this way.
```

Comment:
OOPS keep the ValidTgts - get rid of the TgtPrompt

sorry if I used the wrong param word before